host-bin/enter-chroot

```diff
@@ -528,10 +528,10 @@ if ! mountpoint -q "$CHROOT/sys"; then
     mount --make-rshared /sys
     mount --rbind /sys "$CHROOT/sys"
     mount --make-rslave "$CHROOT/sys"
-    # Remount selinux as ro
-    if [ -d "$CHROOT/sys/fs/selinux" ]; then
```

Comment:
Actually, you still probably want to check for the existence of selinux before unmounting it.